sentry-apache-http-client-5/src/test/kotlin/io/sentry/transport/apache/ApacheHttpClientTransportTest.kt

```diff
@@ -150,22 +158,22 @@ class ApacheHttpClientTransportTest {
     fun `logs warning when flush timeout was lower than time needed to execute all events`() {
         val sut = fixture.getSut()
         whenever(fixture.client.execute(any(), any())).then {
-            CompletableFuture.runAsync {
-                Thread.sleep(100)
+            fixture.executorService.submit {
+                Thread.sleep(1000)
```

Comment:
Can't complain about CI being slow then :P

Reply:
We're waiting up to 200ms (line 174). Here it could be also one minute, the tests checks that the second event is sent and first one discarded.